common/data/dataHelpers.ts

```diff
@@ -1,16 +1,73 @@
 export const STRAPI_BASE_URL = 'https://map-api.ekaterinburg.io/api';
 
+const PAGINATION_SIZE = 800;
+const MAX_REQUESTS = 10;
+
 export async function fetchAPI(url: string) {
     const response = await fetch(url);
     return response.json();
 }
 
-export async function getObjectsTotalCount(urlWithoutParams: string) {
-    return (await fetchAPI(`${urlWithoutParams}?pagination[pageSize]=1`))
-        .meta.pagination.total;
+export async function getObjectsTotalCount(url: string) {
+    const hasParams = url.includes('?');
+    return (await fetchAPI(`${url}${hasParams ? '&' : '?'}pagination[pageSize]=1`)).meta.pagination
```

Comment:
```suggestion
    const requestUrl = (new URL(url)).searchParams.append('pagination[pageSize]', 1);
    return (await fetchAPI(requestUrl)).meta.pagination
```

Reply:
Yeah, thx, I wanted to but forgot about it